backend/Dockerfile

```
@@ -0,0 +1,93 @@
+# =============================================================================
+# STAGE 2: PRODUCTION STAGE
+#
+# Start fresh with a (Minimal Debian).
+# =============================================================================
+
+FROM debian:bookworm-slim
+
+# Install only runtime dependencies (no build tools)
+# ca-certificates: For HTTPS connections
+# curl: needed for healthcheck
+RUN apt-get update && apt-get install -y \
```

Hadolint (2.14.0)

[warning] Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`

(DL3008)